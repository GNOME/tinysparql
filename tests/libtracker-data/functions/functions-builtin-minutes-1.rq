SELECT MINUTES(?d) {
  VALUES ?d {
    '2022-03-10T12:23:34Z'^^xsd:dateTime
    '2022-01-01T01:00:00-0600'^^xsd:dateTime
    '2022-12-31T23:00:00+0600'^^xsd:dateTime
    '0001-01-02'^^xsd:date
    '9998-12-31'^^xsd:date
  }
}
