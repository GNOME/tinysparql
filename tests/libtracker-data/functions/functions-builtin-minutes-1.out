"23"
"0"
"0"
"0"
"0"
